### ISSUE TITLE
pytest --json-fixtures doesn't work without another argument

### ISSUE THREAD

**Issue by ToddBradley** [marcosanchotene/pytest-json-fixtures#1]
The README for your project shows an example command of `pytest --json-fixtures` but this doesn't work. If I do that, it complains that it's expecting another argument. Doing `pytest --json-fixtures foo.json` does work. So I think your handling of command line arguments needs some work.

**Comment by ToddBradley** [comment]
Looking into this more, I can't get it to work right with `pytest --json-fixtures foo.json` either. So something else is wrong. I will close this issue until I can figure out what my *real* problem is.

**Comment by marcosanchotene** [comment]
I'm reopening the issue as it's not working for me also without the file name argument. However, with the file name argument it correctly creates the file. Which version of Python and pytest are you using?

If you cannot wait for a fix, you can try to implement the code yourself, as I explained on this [answer for a Stackoverflow question](https://stackoverflow.com/a/73170331/5489111). Have a look at the main file [here](https://github.com/marcosanchotene/pytest-json-fixtures/blob/main/pytest_json_fixtures.py).

**Comment by ToddBradley** [comment]
Yeah, that Stack Exchange answer is what led me to your project in the first place.

Regarding this bug, I did get this plugin to generate `fixtures.json` files, both with the default name and with a custom name provided on the command line.

I was using Python 3.11 and `pytest-json-fixtures` 0.1.2.

However, I found a couple other bugs, which I'll write up separately. And then I got so frustrated I just decided to use `--fixtures` instead.
